src/raftstore/store/fsm/peer.rs

```diff
@@ -1264,6 +1264,10 @@ impl<T: Transport, C: PdClient> Store<T, C> {
             peer.size_diff_hint = self.cfg.region_split_check_diff.0;
```

Comment:
Should we update `size_diff_hint` on followers?

Reply:
update `size_diff_hint` or `approximate_size` does same effect, it is all about to invoke size check.